tests/network/user_defined_network/test_user_defined_network.py

```diff
@@ -12,14 +12,12 @@
 from libs.vm.affinity import new_pod_anti_affinity
 from libs.vm.factory import base_vmspec, fedora_vm
 from utilities.constants import PUBLIC_DNS_SERVER_IP, TIMEOUT_1MIN
+from utilities.infra import create_ns
 from utilities.virt import migrate_vm_and_verify
 
-# For version 4.18, this module can only run on clusters where FeatureGate is configured with
-# featureSet TechPreviewNoUpgrade.
-pytestmark = pytest.mark.udn
-
 IP_ADDRESS = "ipAddress"
 SERVER_PORT = 5201
+SUCCESS_STATUS = "NetworkAllocationSucceeded"
```

Comment:
Used only once. Please use directly.

Reply:
Used once is fine as long as it provides information, i.e. expresses the meaning of the value.

The problem I see here is that it has no context. Success of what and for what.
That is why I do not like it.